core/src/main/se/tevej/game/input/TMouse.java

```diff
@@ -0,0 +1,32 @@
+package main.se.tevej.game.input;
+
+public interface TMouse {
+
+    TMouse addDraggedListener(OnDraggedListener onDraggedListener);
+
+    TMouse addClickedListener(OnClickedListener onClickedListener);
+
+    TMouse addMovedListener(OnMovedListener onMovedListener);
+
+    float getX();
+
+    float getY();
+
+    interface OnDraggedListener {
+        void onDragged(TMouse mouse, MouseButton button, float x, float y );
```

Comment:
All of these inner interfaces should probably not exist (Same as earlier comment).

Reply:
I am not sure as of now and want to hear from both your side and portals side regarding this issue. I want to merge this pull-request and i have added an issue to fix the inner classes.